tasks/agent-linux.yml

```diff
@@ -66,7 +66,7 @@
     src: system-probe.yaml.j2
     dest: /etc/datadog-agent/system-probe.yaml
     mode: 0640
-    owner: "root"
+    owner: "{{ datadog_user }}"
     group: "{{ datadog_group }}"
```

Comment:
I think it should be the group that changes, not the owner.  The default value for `datadog_group` in ansible is root. 


In chef the owner is root, and the group is datadog-agent.

https://github.com/DataDog/chef-datadog/blob/a3adb45dee5d821a4af860c8d20ced13ad9cc62b/recipes/system-probe.rb#L44-L45

Reply:
Do we need to change the group globally in the playbook or just for the system probe?

@KSerrania do you know why `datadog_group` defaults to `root`?

Reply:
Not really, the user and group were set 4 years ago.  I found this commit: https://github.com/DataDog/ansible-datadog/commit/70b3d5ddc1cfb91f4f5a376f95582b4884e540fb which explains that the group must be root, otherwise the debian package install will fail. However, since this was 4 years ago, we should check if that's still the case. If it is not, aligning the role with our other installation methods may be a good idea.

Reply:
We changed the group on Puppet [1] from `root` to `dd-agent` and so far we have seen no complaints. I think it should be safe to do it everywhere.

[1] https://github.com/DataDog/puppet-datadog-agent/blame/master/manifests/params.pp#L47